### PR TITLE
http proxy middlewere

### PR DESCRIPTION
**Pull request by imhamza333**

### CODE REVIEW

File: settings.py [hunk 1]
```diff
@@ -7,82 +8,109 @@
...
+# HTTPERROR_ALLOWED_CODES = [404]
+# USER_AGENT = 'quotesbot (+http://www.yourdomain.com)'
+# USER_AGENT = "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/34.0.1847.131 Safari/537.36"
+
+
+# PROXY_POOL_ENABLED = True
+
 # Configure maximum concurrent requests performed by Scrapy (default: 16)
-#CONCURRENT_REQUESTS = 32
+# CONCURRENT_REQUESTS = 32
```
Comment (AdnanAsif) [inline settings.py:32]: Why all the settings are commented out ?

File: settings.py [hunk 2]
```diff
@@ -7,82 +8,109 @@
...
+
 
 # Enable or disable downloader middlewares
 # See https://docs.scrapy.org/en/latest/topics/downloader-middleware.html
-#DOWNLOADER_MIDDLEWARES = {
-#    'scraping.middlewares.ScrapingDownloaderMiddleware': 543,
-#}
+DOWNLOADER_MIDDLEWARES = {
+    'scrapy.downloadermiddlewares.httpproxy.HttpProxyMiddleware': 110,
+    'scraping.middlewares.ScrapingDownloaderMiddleware': 543,
```
Comment (AdnanAsif) [inline settings.py]: Where is this file ? I see a class `ProxiesDownloaderMiddleware` in cclonline.py file. That middleware code shouldn't be there.

**Review by AdnanAsif — CHANGES_REQUESTED** [review]

File: settings.py (hunk 2 above)
Comment (imhamza333) [reply settings.py]: i did not create a new file I just enable it from setting add middleware there and then i add middleware class in cclonline.py

File: settings.py (hunk 1 above)
Comment (imhamza333) [reply settings.py:32]: before i was testing in same file

**Pull request merged**